lib/ansible/modules/cloud/docker/docker_config.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/python
+#
+# Copyright 2016 Red Hat | Ansible
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: docker_config
+
+short_description: Manage docker configs.
+
+version_added: "2.8"
+
+description:
+     - Create and remove Docker configs in a Swarm environment. Similar to C(docker config create) and C(docker config rm).
+     - Adds to the metadata of new configs 'ansible_key', an encrypted hash representation of the data, which is then used
+       in future runs to test if a config has changed.
+     - If 'ansible_key' is not present, then a config will not be updated unless the C(force) option is set.
```

Comment:
Since this is an implementation detail, why not merge it with the previous bullet point, so that everything about `ansible_key` is in one paragraph?

Reply:
@felixfontein make sense, resolved. 

I guess I can add some integration tests based on `docker_secret`'s.